1-exercises/C-object-methods/exercise-2.js

```diff
@@ -14,8 +14,7 @@ let myPets = {
   tortoise: "Lord Voldetort",
   rabbit: "Bugs",
 };
-
-let petNames =
+let petNames = (Object.values(myPets));
```

Comment:
You don't need the extra set of parenthesis here (e.g. `Object.values()` instead of `(Object.values())`